codility_training/lessons.lesson03.TimeComplexity.PermMissingElem/NowonLee.js

```diff
@@ -0,0 +1,17 @@
+function solution(A) {
+    // write your code in JavaScript (Node.js 8.9.4)
+    var r = 0
+    A.sort(function(a,b){return a-b})
+    if(A[0] !== 1) return 1;
+    else if(A[A.length-1]  !== A.length+1) return A.length+1;
+    else {
+        for(var i=0;i<A.length-1;i++){

```

Comment:
```suggestion
        for(let i=0;i<len-1;i++){
```

위에서 선언한 len을 이용하면 좋습니다.